endpoints/openrtb2/auction.go

```diff
@@ -645,7 +646,17 @@ func (deps *endpointDeps) validateRequest(req *openrtb_ext.RequestWrapper, isAmp
 		return append(errL, err)
 	}
 
-	if ccpaPolicy, err := ccpa.ReadFromRequestWrapper(req); err != nil {
+	var gpp gpplib.GppContainer
+	if req.BidRequest.Regs != nil && len(req.BidRequest.Regs.GPP) > 0 {
+		gpp, err = gpplib.Parse(req.BidRequest.Regs.GPP)
+		if err != nil {
+			errL = append(errL, &errortypes.Warning{
+				Message:     fmt.Sprintf("GPP consent string is invalid and will be ignored. (%v)", err),
+				WarningCode: errortypes.InvalidPrivacyConsentWarningCode})
+		}
```

Comment:
To match the CCPA and TCF behavior, we should set `req.BidRequest.Regs.GPP` to empty if the string is invalid. Right now, it's not so much ignored.

Reply:
Where do we set the TCF string to empty if it is invalid?

Reply:
We aren't. Should we? I guess it makes sense and would be easier for bidders to digest if they care to inspect it but is there a possibility they would like access to the original value?
I don't see anything in the spec on this subject.

Reply:
I thought we treated invalid GDPR strings the same as we do for CCPA. I'm fine with matching GDPR behavior. We can consider this separately.